load-tests/extractor-timer.js

```diff
@@ -0,0 +1,33 @@
+import { asyncTryCatch } from "./utils.js";
+import { logger } from "./logger.js";
+
+export async function timeExtractor(url) {
+  const startTime = performance.now();
+  let data;
+  do {
+    const { result, error } = await asyncTryCatch(() => fetch(url));
+    if (error) {
+      logger.error(`Error fetching extractor data`, { error, url });
+      return;
```

Comment:
Ewentualnie. Tylko jeśli chcemy, zmiany, które opisałem w komentarzu w main.js:

```suggestion
      break;
```